_posts/2018-11-23-NUnit3TestAdapter-3.11.2.md

```diff
@@ -0,0 +1,7 @@
+NUnit3TestAdapter, version 3.11.2 is released.
+
+See [the release notes](https://github.com/nunit/docs/wiki/Adapter-Release-Notes) for what has changed.  Also see section for release 3.11 which contains the bugs and features. 
+
+[Download nuget package](https://www.nuget.org/packages/NUnit3TestAdapter/3.11.2)

```

Comment:
Should match the capitalization of the brand name (NuGet).

Reply:
Fixed